cve_bin_tool/cve_scanner.py

```diff
@@ -179,19 +180,25 @@ def get_cves(self, vendor: str, product: str, version: str, remarks: Remarks):
             else:
                 self.products_without_cve += 1
             self.all_cves.add(cve_data)
-
-    @staticmethod
-    def openssl_convert(version: str) -> str:
+        else:
+            self.products_without_cve += 1
+            self.logger.debug(
+                f"No CVEs found for {vendor} {product}, version {version}. Is the vendor/product info correct?"
+            )
+            self.all_cves.add(
+                CVEData(vendor, product, version, remarks, [CVE("UNKNOWN", "UNKNOWN")])
+            )
+
+    def openssl_convert(self, version: str) -> str:
         """ pkg_resources follows pep-440 which doesn't expect openssl style 1.1.0g version numbering
         So to fake it, if the last character is a letter, replace it with .number before comparing """
-        if len(version) < 1:
+        if not version:  # if version is empty return it.
             return version
 
-        lastchar = version[len(version) - 1]
-        letters = dict(zip(ascii_lowercase, range(26)))
+        lastchar = version[-1]
 
-        if lastchar in letters:
-            version = f"{version[0: len(version) - 1]}.{letters[lastchar]}"
+        if lastchar in self.ALPHA_TO_NUM:
+            version = f"{version[:-1]}.{self.ALPHA_TO_NUM[lastchar]}"
```

Comment:
What does this have to do with adding an unknown vendor, product case in cve_scanner?  It's not a bad change, and I know it's one you want to do, but it either should have been its own PR or should be part of the PR name so that when it's merged it'll be clear that this change is included.  A good rule of thumb: if you can't find a way to elegantly fit the explanation of the change into a single line, it might be worth splitting it off.  Even if it's tiny.

Reply:
I have created a separate PR for that.